From d4ef1b8ad8444f4077c9335b129e656d9290495e Mon Sep 17 00:00:00 2001
From: Tim Foster <tim.foster@joyent.com>
Date: Wed, 26 Jun 2019 10:45:15 +0100
Subject: [PATCH] TRITON-1772 triton.git should reference 'joyent-repos' in
 repos.json Reviewed by: Trent Mick <trentm@gmail.com> Approved by: Trent Mick
 <trentm@gmail.com>

---
 repos.json | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/repos.json b/repos.json
index 8f058e8..d9fc431 100644
--- a/repos.json
+++ b/repos.json
@@ -99,6 +99,9 @@
                 "meta": true
             }
         },
+        {
+            "name": "joyent-repos"
+        },
         {
             "name": "jsstyle",
             "labels": {
-- 
2.21.0

